README.md

```diff
@@ -1 +1,325 @@
+cd hello_world-rails_react
+code .
+bundle install
+```
+### Usage
+
+To run the project, make sure you configured your database correctly, then execute the following command:
+```sh
+  rails db:create
+  rails db:migrate
+  rails db:seed
+```
+
+After the database is set up, run the following command:
+```
+foreman start -f Procfile.dev
+```
+
+Go to http://localhost:5000/ and see it working.
+
+You can also run this using 2 terminals:
+
+In one, run:
+rails server
+
+In the other, run:
+yarn build --watch
+
+Then check the link of the rails server (it should have changed to http://localhost:3000/)
+
+### Run tests
+
+Not implemented tests
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- AUTHORS -->
+## 👥 Authors <a name="authors"></a>
+
+👤 **Muhammad Aminu Salihu**
+
+- GitHub: [@mohashyne](https://github.com/mohashyne)
+- Twitter: [@muhammadslyhu](https://twitter.com/muhammadsalyhu)
+- LinkedIn: [muhammad-salihu](https://linkedin.com/in/msalyhu)
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Nothing so far
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+I welcome contributions to enhance the functionality and user experience of the Morse_Translator project. If you have any ideas, suggestions, or bug reports, feel free to open an issue or submit a pull request.
+
+If you'd like to contribute to this project, please follow these steps:
+
+1. Fork the repository.
+2. Create a new branch for your feature or bug fix.
+3. Make your changes and commit them with descriptive commit messages.
+4. Push your changes to your forked repository.
+5. Submit a pull request to the main repository, explaining your changes in detail.
+
+Please adhere to the coding conventions and guidelines specified in the project.
+
+Contributions, issues, and feature requests are welcome!
+Feel free to check the [issues page](https://github.com/mohashyne/hello_world-rails_react/issues).
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project give it a star ⭐️
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- FAQ -->
+
+## ❓ FAQ <a name="faq"></a>
+
+- **Can I use the project for any purpose?**
+
+  - Yes, you can use this files for anything you need
+
+- **Do I need to ask for permission?**
+
+  - No need to ask for permission.
+
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+** I would like to thank to Microverse for giving this inspiring project **
+
+<p align="right">(<a href="#hello_world-rails_react">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly fix your MIT link. It is broken.

  ![image](https://github.com/mohashyne/hello_world-rails_react/assets/81468675/3b0c80db-afe8-4cea-aae6-cd4203ec0d23)

Reply:
fixed